src/main/java/picard/sam/CreateSequenceDictionary.java

```diff
@@ -196,6 +259,12 @@ protected int doWork() {
             // read reference sequence one by one and write its metadata
             for (ReferenceSequence refSeq = refSeqFile.nextSequence(); refSeq != null; refSeq = refSeqFile.nextSequence()) {
                 final SAMSequenceRecord samSequenceRecord = makeSequenceRecord(refSeq);
+                //add aliases
+                if(contig2aliases.containsKey(samSequenceRecord.getSequenceName())) {
+                    final Set<String> aliases = contig2aliases.get(samSequenceRecord.getSequenceName());
+                    // "Alternative names is a comma separated list of alternative names"
```

Comment:
Not sure why this is in double quotes? Even if this is quoted from a spec, quotes are unnecessary in a comment like this.